wnfs/Cargo.toml

```diff
@@ -31,6 +31,7 @@ multihash = "0.18"
 once_cell = "1.16"
 proptest = { version = "1.1", optional = true }
 rand_core = "0.6"
+rand = "0.8.4"
```

Comment:
What is this needed for?

Reply:
the `get_random_bytes` test case in `wnfs-common/src/utils.rs` relies on this library, but it is the only function that I believe does so. I'm apathetic to its presence, this test case existed before I got here, so I'm unsure how it was compiling without `rand`.

Reply:
`rand` is part of `[dev-dependencies]`. So I think that'll make it available for tests, without being a transitive dependency for users.

Reply:
oh wait, this file is in wnfs/. rand was already a main dependency in wfns/ so this can be removed safely.

Reply:
Too many files! I got confused as well lol